TOPdeskPS/functions/Change Management/New-TdChange.ps1

```diff
@@ -51,7 +51,6 @@
         [system.string]
```

Comment:
let's update the help on this.

Reply:
still some stuff to do, but help is fixed